gammapy/datasets/map.py

```diff
@@ -1841,29 +1842,35 @@ def cutout(self, position, width, mode="trim", name=None):
             Cutout map dataset.
         """
         name = make_name(name)
-        kwargs = {"gti": self.gti, "name": name, "meta_table": self.meta_table}
-        cutout_kwargs = {"position": position, "width": width, "mode": mode}
+        kwargs = {
+            "gti": self.gti,
+            "name": name,
+            "meta_table": self.meta_table,
+            "meta": self.meta,
+        }
+        kwargs["meta"].creation = CreatorMetaData()
```

Comment:
In general, I wonder whether, for data products created by gammapy, the creation metadata could be kept to None until serialization.

Reply:
The metadata propagation should be made easely possible at any step of the analysis. If at each maker, classes, etc, the user should say `meta_propagation=True`, this will be a very inconvenient API. 
However, the feature to switch on and off the propagation is interesting.
I propose to set up a global Gammapy variable, such as the progress bas, to permit this feature... Then, we can expose this variable into the HowTo

Reply:
Otherwise, indeed, the metadata creation should be set-up at the serialisation step

Reply:
The point of adding the 'creation' here is that then it will signify the creation of the cutout itself. 

So instead I should just move the creator to the write functionality and therefore on the MapDataset in general?